discovery/roles/postscripts/common/templates/omnia_gaudi.j2

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
@Aditya-DP  @abhishek-sa1  is this aligning to our implementation?

Reply:
@priti-parate We have already implemented by our team and needs to be reverted

Reply:
@priti-parate, This was requested by you in the past. @abhishek-sa1 Can you provide the link to the PR your team is working on? I don't see any PR or branch regarding this. We shouldn't have multiple work streams. Let's sync Tomorrow in our Omnia meeting to close on this.

Reply:
The script is intended as a reference and can be overwritten by a subsequent PR